pegasus/sites.v3/code.org/public/help.haml

```diff
@@ -33,23 +33,6 @@ theme: responsive_full_width
       %a.link-button{href: "/international/apply"}
         =hoc_s(:call_to_action_become_partner_intl)
 
-%section
-  .wrapper
-    %div.flex-container.justify-space-between.wrap{style: "gap: 3rem"}
-      .content-wrapper.col-45
-        %img.rounded-corners{src: "/images/help-page-volunteer.png", alt: "", style: "width: 100%; margin-bottom: 1.5rem"}
-        %h2=hoc_s(:help_page_volunteer_heading)
-        %p=hoc_s(:help_page_volunteer_desc)
-        %a.link-button{href: "/volunteer"}
-          =hoc_s(:call_to_action_become_a_volunteer)
-      .content-wrapper.col-45{style: "float: right"}
-        %img.rounded-corners{src: "/images/help-page-translate.png", alt: "", style: "width: 100%; margin-bottom: 1.5rem"}
-        %h2=hoc_s(:help_page_translate_heading)
```

Comment:
This page is likely to be replaced by https://code.org/donate which has a call to action for volunteering to translate so I removed this whole section.